src/Stats/Health.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace WPML\ElasticPress\Stats;
+
+use ElasticPress\Indexables;
+use ElasticPress\Utils as Utils;
+
+use WPML\ElasticPress\Manager\Indices;
+
+class Health {
+
+	/** @var Indexables */
+	private $indexables;
+
+	/** @var bool */
+	private $networkActivated;
+
+	/** @var Indices */
+	private $indicesManager;
+
+	/** @var array */
+	private $activeLanguages;
+
+	/** @var string */
+	private $defaultLanguage;
+
+	/**
+	 * @param Indexables $indexables
+	 * @param bool       $networkActivated
+	 * @param Indices    $indicesManager
+	 * @param array      $activeLanguages
+	 * @param string     $defaultLanguage
+	 */
+	public function __construct(
+		Indexables      $indexables,
+		$networkActivated,
+		Indices         $indicesManager,
+		$activeLanguages,
+		$defaultLanguage
+	) {
+		$this->indexables       = $indexables;
+		$this->networkActivated = $networkActivated;
+		$this->indicesManager   = $indicesManager;
+		$this->activeLanguages  = $activeLanguages;
+		$this->defaultLanguage  = $defaultLanguage;
+	}
+
+	public function addHooks() {
+		add_filter( 'ep_index_health_stats_indices', [ $this, 'includeIndicesInHealthStats' ], 10, 2);
```

Comment:
```suggestion
		add_filter( 'ep_index_health_stats_indices', [ $this, 'includeIndicesInHealthStats' ], 10, 2 );
```

@decodekult, even if we might not have installed our linting tools on this project, perhaps your can run PHPCS locally (from another project) to detect this kind of simple-to-fix formatting issues.